components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/impl/WorkersApiServiceImpl.java

```diff
@@ -1105,7 +1106,8 @@ public Response getComponentHistory(String workerId, String appName, String comp
             StatusDashboardMetricsDBHandler metricsDBHandler = WorkersApi.getMetricStore();
             long timeInterval = period != null ? parsePeriod(period) : Constants.DEFAULT_TIME_INTERVAL_MILLIS;
             Map<String, List<List<Object>>> componentHistory = new HashMap<>();
-            if (timeInterval <= 3600000) {
```

Comment:
3600000 - move this as a constant

Reply:
Fixed in 84eed0f2066c8da5780a1ab6e5bbed953ff210b9